api/src/main/java/com/graphhopper/util/Helper.java

```diff
@@ -318,10 +318,11 @@ public static double keepIn(double value, double min, double max) {
     }
 
     /**
-     * Round the value to the specified exponent
+     * Round the value to the specified number of decimal places, i.e. decimalPlaces=2 means we round to two decimal
+     * places. Using negative values like decimalPlaces=-2 means we round to two places before the decimal point.
```

Comment:
Maybe this is what confused me, does this make it more clear?

Reply:
Yes